src/components/GameClock.hpp

```diff
@@ -74,13 +82,13 @@ namespace REGoth
 
     void setTime(bs::UINT32 day, bs::UINT8 hour, bs::UINT8 min);
 
-  /************************************************************************/
-  /* RTTI                                                                 */
-  /************************************************************************/
+    /************************************************************************/
+    /* RTTI                                                                 */
+    /************************************************************************/
   public:
     REGOTH_DECLARE_RTTI(GameClock)
 
-  // protected:
+    // protected:
```

Comment:
What's up with the weird formatting (comments getting indented)? Or is it just following the rules of the format file?

Reply:
I assume Clang-format thinks the comment is documenting the next class member and thus it should be indented. I actually didn't notice that this class still uses the old workaround. Will fix it while I'm at it.